### PR TITLE
Astropy runtime

### PR DESCRIPTION
**Pull request by cameron-goddard**
### Summary

This PR addresses Jira tickets [CISLUNAR-269](https://jira.cornell.edu/browse/CISLUNAR-269) and [CISLUNAR-315](https://jira.cornell.edu/browse/CISLUNAR-315)      

Andrew and I pair programmed on this. We rounded the timesteps to the 10s place to increase the runtime from roughly 46 seconds down to about 1.5 seconds. Similarly, the percentage of the runtime that get_body_position uses decreased from about 98% to about 59%.

### Testing

We ran the profiler on both the unmodified and modified code for freefall. Below are the SnakeVizs for both:

image

image

### Notes

Also fixed the docstring comment on get_body_position per Stephen's announcement at the software meeting.

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

File: src/core/models/model_list.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def d_state(self, t: float, state: State) -> Dict[str, State_Type]:
         # craft to origin
         r_co = np.array([state.x, state.y, state.z])
         # moon to origin
+        
+        # round t to the 10s place
```
Comment (tmf97) [inline src/core/models/model_list.py:44]: Where does this happen?

**Review by tmf97 — CHANGES_REQUESTED** [review]
The snakeviz results look beautiful. But it looks like you forgot to git add or push some changes - I can't see the code that does the 10 sec rounding.
Also, let's delete the .prof file

**Comment by tmf97** [comment]
I'm still a little worried that this one line of imported code is using up >50% of the sims runtime. I'd expect it to use up much less, 5% max. More realistic sim runs in the future will be simulating weeks or even months of sim time, not just 20 minutes. There's gotta be a better way to do this, but this works so 🤷

File: src/core/models/model_list.py (hunk 1 above)
Comment (cameron-goddard) [reply src/core/models/model_list.py:44]: Yeah sorry, I did a clown move and deleted the changes temporarily to compare with the original runtime, but then forgot to change them back when I did a commit after. Should be there now though

**Review by tmf97 — CHANGES_REQUESTED** [review]
Looks solid! Just remove the for_snakeviz.prof file and we're good!

**Comment by andrew032011** [comment]
Done!

**Review by tmf97 — APPROVED** [review]
E P I C

**Review by sjz38 — APPROVED** [review]
👍Yup looks good. Great performance improvement. Hopefully, we can manage with this performance

**Pull request merged**